builder/src/extensionConfig.ts

```diff
@@ -235,6 +235,13 @@ function generateConfig({
     );
   }
 
+  // Add version argument when in production so the Jupyter server
+  // does not cache the files in the static file handler.
```

Comment:
From the Jupyter server code, it appears that the effect is the opposite - adding the `v=` argument makes it so that Jupyter server does not set the no-cache header, i.e., that the files *are* cached. Is that right?